### DISCUSSION TITLE
Entity Hider - Hide Blocked Players

### DISCUSSION THREAD

**Discussion by acory32590** [runelite/runelite#15788] [Q&A] [answered]
As title suggests.  Can we get an option to hide players on the blocked list?  Useful if people are trying to crash events, even just by physically be around

**Comment by Alexsuperfly** [comment, marked as answer]
already exists `Hide Ignores`

**Reply by acory32590** [reply]
Oh my god, I was so tunnel visioned on looking for keyword "blocked" i glossed over the Hide Ignores option.  Thanks a lot!
